kodilla-exception/src/main/java/com/kodilla/exception/test/FlightFinder.java

```diff
@@ -0,0 +1,24 @@
+package com.kodilla.exception.test;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+public class FlightFinder {
+
+    public Optional findFlight(Flight flight) throws RouteNotFoundException {
```

Comment:
```Optional<Flight>```

Reply:
Optional.of(x).  -- opakuj w pudelko tylko wtedy, gdy x nie moze byc nullem
Optional.ofNullable(x). -- tutaj x moze byc nullem => if (x == null) return Optional.empty(); else return Optional.of(x);
Optional.empty() -- puste pudelko